### PR TITLE
Added comment for Orbit Yaw Behavior enum.

### PR DESCRIPTION
**Pull request by DoppleGangster**
Side note the enum in the documentation has behavior spelled as behaviour for Orbit_Yaw_Behaviour [here](https://mavlink.io/en/messages/common.html).

### CODE REVIEW

File: protos/action/action.proto [hunk 1]
```diff
@@ -185,6 +185,7 @@ message GotoLocationResponse {
     ActionResult action_result = 1;
 }
 
+// Yaw behaviour during orbit flight.
 enum OrbitYawBehavior {
     ORBIT_YAW_BEHAVIOR_HOLD_FRONT_TO_CIRCLE_CENTER = 0;
```
Comment (JonasVautherin) [inline protos/action/action.proto:190]: It's missing the documentation next to the values, like [here](https://github.com/mavlink/MAVSDK-Proto/blob/main/protos/camera/camera.proto#L218-L222)

**Review by JonasVautherin — CHANGES_REQUESTED** [review]

File: protos/action/action.proto (hunk 1 above)
Comment (DoppleGangster) [reply protos/action/action.proto]: It has been enhanced.

**Comment by DoppleGangster** [comment]
Should be good to go now.

**Review by JonasVautherin — APPROVED** [review]
Thanks!

**Pull request merged**